arbnode/transaction_streamer.go

```diff
@@ -1558,6 +1542,53 @@ func (s *TransactionStreamer) HasNotSubmitted(pos arbutil.MessageIndex) (bool, e
 	return true, nil
 }
 
+/**
+* If the transaction fails to fetch, we will retry for a while before resubmitting
+* the transaction to the Espresso node.
+ */
+func (s *TransactionStreamer) tryResubmittingEspressoTransactions(ctx context.Context, firstSubmitted SubmittedEspressoTx, submittedTxHash *tagged_base64.TaggedBase64, err error) error {
+	if s.lastSubmitFailureAt == nil {
+		now := time.Now()
+		s.lastSubmitFailureAt = &now
+		return fmt.Errorf("trying to resubmit transaction (hash: %s): %w, will retry", submittedTxHash.String(), err)
```

Comment:
Nit: I think this error could be a bit misleading. We're not trying to resubmit (yet), we're still just waiting to see if it gets included or not since this would happen only the first time we fail to poll for finality

Reply:
i'll update it to be the same message as the one when not enough time has passed